File: filedemo/src/main/java/com/awesome/filedemo/MainActivity.java
package com.awesome.filedemo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.cocav.tiemu.utils.opus;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        opus.getInstance();
    }
}

File: hiltdemo/src/main/java/com/awesome/hiltdemo/MainActivity.kt
package com.awesome.hiltdemo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import dagger.hilt.android.AndroidEntryPoint
import javax.inject.Inject

@AndroidEntryPoint
class MainActivity : AppCompatActivity() {
    @Inject
    lateinit var truck: Truck
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        truck.deliver()
    }
}